Makefile

```diff
@@ -161,26 +163,36 @@ install: install-dirs
 		 experiments/powertop2tuned.py, $(DESTDIR)/usr/bin/powertop2tuned)
 
 	# configuration files
-	install -Dpm 0644 tuned-main.conf $(DESTDIR)$(SYSCONFDIR)/tuned/tuned-main.conf
+	# Update profiles location
+	sed '/^\s*#\?\s*profile_dirs\s*=\s*/ s|\(profile_dirs\s*=\s*\).*$$|\1$(TUNED_SYSTEM_PROFILES_DIR),$(TUNED_USER_PROFILES_DIR)|' \
+		tuned-main.conf > tuned-main.conf.tmp
+	touch -r tuned-main.conf tuned-main.conf.tmp
+	sed -i '/^\s*USER_PROFILES_DIR\s*=\s*/ s|\(USER_PROFILES_DIR\s*=\s*\).*$$|\1"$(TUNED_USER_PROFILES_DIR)"|' \
+		$(DESTDIR)$(PYTHON_SITELIB)/tuned/consts.py
+	sed -i '/^\s*SYSTEM_PROFILES_DIR\s*=\s*/ s|\(SYSTEM_PROFILES_DIR\s*=\s*\).*$$|\1"$(TUNED_SYSTEM_PROFILES_DIR)"|' \
+		$(DESTDIR)$(PYTHON_SITELIB)/tuned/consts.py
+	install -Dpm 0644 tuned-main.conf.tmp $(DESTDIR)$(TUNED_CFG_DIR)/tuned-main.conf
+	rm -f tuned-main.conf.tmp
```

Comment:
Can you please add a comment here, explaining why it's necessary? Took me quite a while to figure out what's happening here.

I'm not a big fan of modifying the code like this, but I guess it's the best option at this point. Another way would be to add new options into `tuned-main.conf`, specifying the system/user profile directories. But since we already have the `profile_dirs` option there, it could get confusing...

Reply:
Yep, probably better ways are usage of the m4 macro pre-processor or template file. It could also include generated file, but I didn't want to complicate things just for two variables.

Reply:
I improved the comment, but feel free to propose some more text.